app/controllers/admin/orders_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Admin::OrdersController < ApplicationController
-  before_action :is_admin?
+  load_and_authorize_resource
```

Comment:
```suggestion
  authorize_resource
```
cần load ko em?
load trong method load_and_authorize_resource nghĩa là gì em, nó có chức năng gì nào

Reply:
Dạ load để tạo ra dữ liệu trước khi kiểm tra quyền truy cập các action. Có thể khởi tạo đối tượng hoặc get theo params. Trong action đã có các bước để lấy dữ liệu nên ko cần load nữa ạ.